### PR TITLE
Port Step 2 of "Add Safe" dialog

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
### Testing time!
- Check that all the link works as expected.
- Check that the read-only values make sense.
- Check that the module contract address gets validated successfully both when it's the correct and when it's the wrong address.
- I tested this with these values: `0xe759f388c97674D5B8a60406b254aB59f194163d` (Binance Safe) and `0xCC199aCAb2B38C6DC5f7304B29BDEb3a83E7d212` (bridge module)

FireShot Capture 116 - Colony CDapp - localhost

Resolves #707

### CODE REVIEW

**Review by alicjakujawa — APPROVED** [review]

**Review by thomassdk — COMMENTED** [review]
I noticed one thing where the values of the `Controller Contract Address` is different between the colony dialog and the safe dialog 

Screenshot from 2023-07-11 13-06-34
Screenshot from 2023-07-11 13-06-13

is this an issue?

EDIT: I also second Alicija's comment that the magic number `1` is a little unclear

**Comment by ArmandoGraterol** [comment]
@thomassdk 
> is this an issue?

Nope, they are different because that bridge module was initialized for a QA colony, however, for local dev env that doesn't matter as we can't send "real" transactions to safes anyway (we deploy mockups of the Safe, bridge module, and AMBs to simulate sending Safe transactions).

It would matter if we were in QA or prod, as the controller should be the same address as the colony we use to interact with the safe.

File: src/components/common/Dialogs/AddExistingSafeDialog/AddExistingSafeDialogForm.tsx [hunk 1]
```diff
@@ -70,16 +49,14 @@ const AddExistingSafeDialogForm = ({
...
-      //       safeAddress={contractAddress}
-      //       loadingState={loadingState[1]}
-      //     />
-      //   );
+      case 2:
+        return (
+          <ConnectSafe
+            {...props}
+            setStepIndex={setStepIndex}
+            loadingState={loadingState[1]}
```
[the remark this answers is not in the document]
Comment (ArmandoGraterol) [reply src/components/common/Dialogs/AddExistingSafeDialog/AddExistingSafeDialogForm.tsx:57]: I agree! Updated

**Review by thomassdk — APPROVED** [review]
Great work

**Pull request merged**